src/whisper_at_train/run_combined_training.sh

```
@@ -6,12 +6,25 @@
+#SBATCH --time=48:00:00
 
 set -x
+set -e  # Exit on any error
+
 # comment this line if not running on sls cluster
 #. /data/sls/scratch/share-201907/slstoolchainrc
-source /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate
+
+# Activate virtual environment
+if [ -f "/home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate" ]; then
+    source /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate
```

Shellcheck (0.10.0)

[info] Not following: /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate was not specified as input (see shellcheck -x).

(SC1091)